web-library/Book/Request/AssigneGenreToBookRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Text.Json.Serialization;
+
+namespace web_library.Book.Request
+{
+    public class AssigneGenreToBookRequest
+    {
+        
+        public ICollection<int> genres_ids { get; set; }
+        [JsonIgnore] 
```

Comment:
Po co to ?

Reply:
request do kontrolera książki który do isniejącej książki dodaje gatunki